utils/monitor.js

```diff
@@ -49,9 +49,9 @@ const buildQueryForFetchingDocsOfType = (queryParams) => {
 const buildQueryToFetchTrackedDoc = (queryParams) => {
   const { userId, taskId } = queryParams;
   if (userId) {
-    return trackedProgressesCollection.where("userId", "==", userId);
+    return trackedProgressesCollection.where("type", "==", "user").where("userId", "==", userId);
   } else {
-    return trackedProgressesCollection.where("taskId", "==", taskId);
+    return trackedProgressesCollection.where("type", "==", "task").where("taskId", "==", taskId);
```

Comment:
If the two functions have the very same piece of code, why are there two functions(`buildTrackedProgressQueryByType`, `buildQueryToFetchTrackedDoc`) with a different name rather than one with a generic name(`buildQuery`)?

This is a question, not code change haha

Reply:
Right. I might have created it unintentionally. I will. remove one of them. Thanks for finding this one.